### ISSUE TITLE
training loss becomes NaN after few epochs for large model

### ISSUE THREAD

**Issue by avs6194** [ahmetumutdurmus/zaremba#1]
After a few training epochs( around 5/6) for the large LSTM model, the network loss becomes NaN .. Is this problem known ? I'm using recommended software environment on a RedHat cluster.
